content/docs/reference/autocert/readme.md

```diff
@@ -19,6 +19,10 @@ Turning on autocert allows Pomerium to automatically retrieve, manage, and renew
 
 This setting can be useful in situations where you may not have Pomerium behind a TLS terminating ingress or proxy that is already handling your public certificates on your behalf.
 
+Autocert will incorporate certificates available in the system trust store, and they will take precedence over generated certificates when applicable to configured routes.
```

Comment:
This is true of manually specified certs too  (https://www.pomerium.com/docs/reference/certificates)

```suggestion
Autocert will incorporate certificates available in the system trust store and those set manually as pomerium configuration, and they will take precedence over generated certificates when applicable to configured routes.
```

Maybe better way of phrasing it possible?

Reply:
```suggestion
Autocert will incorporate certificates available in the system trust store and those set manually in the Pomerium configuration, and they will take precedence over generated certificates when applicable to configured routes.
```

Reply:
LGTM!